### DISCUSSION TITLE
Fix "Always release services not following semantic versioning" Feature

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#214] [Ideas]
Hi! When deploying, I find the "Always release services not following semantic versioning" checkbox label to be misleading, as it does not indeed _always_ release services not following semantic versioning, but it releases those services only for the upcoming deploy. Am I missing something? I think that this feature should either be renamed to something like "release services not following semantic versioning" (omitting the "Always" adverb"), or persist its value in the user's browser so that when you activate this feature it remains active for the following deploys (at least on the same project and for the same environment)

![image](https://user-images.githubusercontent.com/21307773/226370078-03c131ba-2947-41e7-b94f-45395961084f.png)

**Comment by fredmaggiowski** [comment]
Hi @GioAc96 thank you for your feedback, we could change the copy for sure; I'd avoid saving the information to prevent unintended errors during deploy

 @albertotessarotto @jacoposte FYI

**Reply by GioAc96** [reply]
What about allowing persisting this option only for non production environments? Also, we could start by simply changing the copy and later adding a separate "remeber this choice for the current environment" checkbox below it, if we find this feature to be desireable.

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
